### PR TITLE
Adc 500219 web app documentation

### PR DESCRIPTION
**Pull request by NickSwainston**
Added documentation on how to use the web application including
- How to make a proposal
- How to add a user
- Trigger logic explanation

The docs can be viewed here:
https://mwa-trigger.readthedocs.io/en/adc-500219-web-app-documentation/

### CODE REVIEW

File: docs/triggerservice_module.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ triggerservice Module
 
 .. currentmodule:: mwa_trigger.triggerservice
 
+trigger_mwa Function
+--------------------
+.. autofunction:: trigger_mwa
```
Comment (PaulHancock) [inline docs/triggerservice_module.rst:9]: these `autofunction` directives don't seem to generate any content.

File: docs/installation.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+   sudo -u Postgres psql
+
+   CREATE DATABASE trigger_db;
+   CREATE USER $DB_USER WITH ENCRYPTED PASSWORD "$DB_PASSWORD";
+
+   ALTER ROLE $DB_USER SET client_encoding TO 'utf8';
+   ALTER ROLE $DB_USER SET default_transaction_isolation TO 'read committed';
+   ALTER ROLE $DB_USER SET timezone TO 'UTC';
+
```
Comment (PaulHancock) [inline docs/installation.rst:50]: Could you also add instructions on how to clear the DB and start over? Probably not something that needs to be done that often (except when I'm testing new PR).

**Review by PaulHancock — CHANGES_REQUESTED** [review]

**Comment by PaulHancock** [comment]
The "start postgres database section" of `installation.rst` needs some adjustments:
```
sudo -u postgres psql   # <- lower case p

CREATE DATABASE trigger_db;
CREATE USER $DB_USER WITH ENCRYPTED PASSWORD '$DB_PASSWORD';  # <- single quotes

ALTER ROLE $DB_USER SET client_encoding TO 'utf8';
ALTER ROLE $DB_USER SET default_transaction_isolation TO 'read committed';
ALTER ROLE $DB_USER SET timezone TO 'UTC';
```

File: docs/triggerservice_module.rst (hunk 1 above)
Comment (NickSwainston) [reply docs/triggerservice_module.rst:9]: There was an issue with the installation of the dependencies which I've fixed now (you may need to use crtl f5 to see the updates)

File: docs/installation.rst (hunk 2 above)
Comment (NickSwainston) [reply docs/installation.rst:50]: Done

**Comment by PaulHancock** [comment]
`twilio` is missing from the `requirements.txt` file

**Comment by NickSwainston** [comment]
> The "start postgres database section" of `installation.rst` needs some adjustments:
> 
> ```
> sudo -u postgres psql   # <- lower case p
> 
> CREATE DATABASE trigger_db;
> CREATE USER $DB_USER WITH ENCRYPTED PASSWORD '$DB_PASSWORD';  # <- single quotes
> 
> ALTER ROLE $DB_USER SET client_encoding TO 'utf8';
> ALTER ROLE $DB_USER SET default_transaction_isolation TO 'read committed';
> ALTER ROLE $DB_USER SET timezone TO 'UTC';
> ```

Fixed

**Comment by NickSwainston** [comment]
> `twilio` is missing from the `requirements.txt` file

Done (tried to remove obvious bloat)

**Pull request merged**